rubygem/spec/zeus_spec.rb

```diff
@@ -0,0 +1,85 @@
+require 'zeus'
```

Comment:
This test involves some fiddly things with IOs and concurrency... probably worth a careful read.